plugins/crypto/mbedtls/ua_certificategroup_memorystore_mbedtls.c

```diff
@@ -0,0 +1,494 @@
+/* This work is licensed under a Creative Commons CCZero 1.0 Universal License.
+ * See http://creativecommons.org/publicdomain/zero/1.0/ for more information.
+ *
+ *    Copyright 2024 (c) Fraunhofer IOSB (Author: Noel Graf)
```

Comment:
Also add the people who worked on the original implementation.
A lot of the code was reused. The same for the OpenSSL implementation.